mlflow/server/js/src/experiment-tracking/components/MetricsPlotPanel.js

```diff
@@ -459,6 +498,18 @@ export class MetricsPlotPanel extends React.Component {
     }, 300);
   };
 
+  checkOnRunUnfinished() {
+    const { runs } = this.props;
+
+    let boolean = false;
+    runs &&
+      runs.forEach((run) => {
+        if (run.getStatus() === 'RUNNING') boolean = true;
+      });
+
+    return boolean;
+  }
+
```

Comment:
```suggestion
  checkOnRunUnfinished() {
    const { runs } = this.props;

    let boolean = false;
    runs &&
      runs.forEach((run) => {
        if (run.getStatus() === 'RUNNING') boolean = true;
      });

    return boolean;
  }

```

Can we remove this function since it's no longer used?

Reply:
Sure